nixos/tests/podman-apparmor.nix

```diff
@@ -0,0 +1,63 @@
+# This test runs podman and checks AppArmor default confinement
+
+import ./make-test-python.nix (
+  { pkgs, lib, ... }: {
+    name = "podman";
+    meta = {
+      maintainers = lib.teams.podman.members;
```

Comment:
Please add yourself as maintainer here or as a member of the `podman` team.

Reply:
Feels maybe too ambitious to add myself to the team (yet), so I just added myself to this test.